src/main/java/it/pn/frontend/e2e/common/NotificheDestinatarioPage.java

```diff
@@ -37,4 +38,10 @@ public boolean verificaCodiceIUN(String codiceIUNInserito) {
         return false;
     }
 
+    public boolean isTextBoxInvalid(){
+        getWebDriverWait(30).withMessage("L'input codice IUN non è visibile").until(ExpectedConditions.visibilityOf(codiceIunTextField));
+        String ariaInvalid = codiceIunTextField.getAttribute("aria-invalid");
+        return "true".equals(ariaInvalid);
```

Comment:
inserisci la stringa true in un valore booleano final

Reply:
grazie per il feedback, ho fatto le modifiche, ora faccio push